streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java

```diff
@@ -153,7 +154,7 @@ public void testJoin() throws Exception {
 
         KStream<String, String> stream1 = builder.stream(strSerde, strSerde, topic1);
         KTable<Windowed<String>, String> table1 =
-            stream1.groupByKey(strSerde, strSerde)
+            stream1.groupByKey(Serialized.with(strSerde, strSerde))
```

Comment:
Can we format this `Serialized.with` like the one above?

Reply:
i'll just make them like this one as it fits comfortably on a single line